### DISCUSSION TITLE
Horizontal offset

### DISCUSSION THREAD

**Discussion by dmbaranov** [Akryum/v-tooltip#448] [Q&A] [answered]
Hello. I'm trying to add offset for the tooltip on X axis. I tried to achieve it using directive's `offset` option and `popperOptions` as well, neither of those worked. Please take a look at this example. I'd like tooltip to be shifted to the left or right for 30px. Thanks in advance!

https://codesandbox.io/s/interesting-fast-t22t4

**Comment by billmn** [comment]
I'm also interested on how to set a horizontal offset to a popover.

I have a popover on the right side of a navbar and I would translate the "popover-inner" some px to the left to prevent it from falling under the navigation bar.

![Schermata 2020-04-30 alle 10 49 48](https://user-images.githubusercontent.com/779534/80691760-fbc45480-8ad0-11ea-990a-585bf817527c.png)

Unfortunately I don't understand how to do it.

**Comment by dmbaranov** [comment]
@billmn while there's no fix for this issue, I temporarily solved it by creating an additional span and moving it around as I want. Hope that helps you as well!

**Comment by billmn** [comment]
@dmbaranov  I did something similar as a workaround too.

Thank you for the suggestion

**Comment by stuartromanek** [comment, marked as answer]
it is possible to do this, however v-tooltip implements it a bit differently than Popper.js does under the hood.

You can provide a comma separated string as the offset in an `x,y` format

`:offset="'100, 20'"`

Should go 100px left, 20px down
